en/04_Changelogs/5.2.0.md

```diff
@@ -77,6 +78,16 @@ $query->selectField('"my custom title" AS "Title"');
 $query->setAllowCollidingFieldStatements(true);
 ```
 
+### GridField components now work with arbitrary data {#gridfield-arbitrary-data}
+
+It has historically been difficult to use a `GridField` to display data that isn't represented by `DataObject` records - and even more difficult to edit that data.
+
+We have removed several barriers to using the `GridField` to display arbitrary data, and improved error messaging when specific information cannot be dynamically identified, such as which columns to display and what form fields to use when viewing or editing data.
```

Comment:
```suggestion
We have removed several barriers to using the `GridField` to display arbitrary data. Descriptive exceptions will be thrown when specific information cannot be dynamically identified, such as which columns to display and what form fields to use when viewing or editing data.
```

Reply:
Done but that's meant I need to add a note that these aren't bc breaking exceptions.